src/esp/core/CMakeLists.txt

```diff
@@ -22,6 +22,10 @@ if(BUILD_WITH_VHACD)
   set(ESP_BUILD_WITH_VHACD ON)
 endif()
 
+if(BUILD_WITH_AUDIO)
```

Comment:
QUESTION:
Im assuming this is the incorrect place to enable the flag. Should this be set in sensor's cmakelists.txt?